lib/modules/manager/gomod/update.ts

```diff
@@ -22,7 +22,7 @@ export function updateDependency({
     // newName will be available for replacement
     const toPackageName = newName ?? fromPackageName;
     // istanbul ignore if: should never happen
-    if (!fromPackageName || !toPackageName || !upgrade.managerData) {
+    if (!fromPackageName || !upgrade.managerData) {
```

Comment:
we may need to undo this check to ensure that `eslint` knows that we've checked this